components/brave_rewards/resources/rewards_panel/lib/extension_api_adapter.ts

```diff
@@ -199,10 +199,6 @@ let grantPromise: Promise<GrantInfo[]> | null = null
 let grantsUpdatedCallbacks: GrantsUpdatedCallback[] = []
 
 chrome.braveRewards.onPromotions.addListener((result, promotions) => {
-  if (result === 1) { // Error
-    return
-  }
-
   const grants: GrantInfo[] = []
```

Comment:
The error handling branch above was removed, since in that case `promotions` will be an empty list and we want to send that empty list to handlers.